dlblob.js

```diff
@@ -21,7 +21,7 @@
 'use strict';
 
 const async = require('async');
-const azure = require('azure');
+const azure = require('azure-storage');
```

Comment:
are you sure `azure-storage` has `createBlobService ()` function?

Reply:
I am reasonably sure. The tests pass after an npm install.